### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Akhtar-Joomun**
Files added:
- Index.html
- style.css
- Images used for the design and functionality
- A screenshot of the webpage
- README.md file with the important information required
- Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
...667 10.355C2.93001 10.0542 3.43167 9.53083 3.73834 9.53083ZM2.39834 12.5892C2.85167 12.5892 4.55917 14.945 4.55917 15.445C4.55917 15.6367 4.2525 15.7692 4.09417 15.7692C3.45334 15.7692 3.2025 15.1633 2.06084 13.7617C0.878338 14.9217 2.87667 17.1558 4.38167 17.1558C5.42001 17.1558 6.09667 16.2108 5.89417 15.515C6.04167 15.515 6.22417 15.5267 6.36 15.4917C6.40417 16.55 6.7225 17.8117 8.02001 17.9017C8.02001 17.8667 8.10001 17.6242 8.10001 17.6125C8.10001 16.9325 7.67834 16.3392 7.67834 15.6475C7.67834 14.5417 8.54167 13.4717 9.41751 12.8467C9.73584 12.6125 10.1225 12.4675 10.4958 12.335C10.8825 12.1908 11.2925 12.0225 11.5858 11.7333C11.5417 11.2958 11.3592 10.9092 10.9133 10.9092C9.81084 10.9092 6.11334 11.0658 6.11334 9.35833C6.11334 9.09667 6.11751 8.84667 6.80667 8.84667C8.09251 8.84667 11.3567 9.15917 12.3117 9.98333C13.0317 10.6125 13.2792 14.405 11.0775 16.8075L11.075 16.8083C9.91667 18.0583 8.43584 18.7067 6.70834 18.7067C4.34417 18.7067 2.47667 17.4333 1.5775 15.2575C0.896672 13.5625 1.72917 12.5892 2.39834 12.5892ZM7.14917 11.8858C7.53667 12.0075 7.93417 12.0433 8.33251 12.1208C8.03751 12.3317 7.77584 12.59 7.52417 12.8667C7.41251 12.535 7.27667 12.2108 7.14917 11.8858Z" />
```
Comment (DeliceLydia) [inline index.html]: - [x] It is good to see that you use ul and li tags but you need to make each list item a link kindly fix that

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                </svg>
+            </li>
+            <li class="list-item">
+                <svg width="20" height="16" viewBox="0 0 20 16" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z" />
+                </svg>
+            </li>
+        </ul>
+        <img class="illus" src="illustration.png" alt="" />
```
Comment (DeliceLydia) [inline index.html]: - [x] For the section background you are supposed to use background image not img tag please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Ak0706, 

Good job so far! your project looks awesome already but there is always room for improvements. There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Akhtar-Joomun) [reply index.html]: Links have been added to the icons.

File: index.html (hunk 2 above)
Comment (Akhtar-Joomun) [reply index.html]: Background image has been added instead of image tags.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi @Ak0706 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**